wrangler-service/src/main/java/co/cask/wrangler/service/spanner/SpannerInstance.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.wrangler.service.spanner;
+
+/**
+ * Spanner instance information
+ */
+public class SpannerInstance {
+  private final String instanceName;
```

Comment:
is the existing pattern to call these xyzName? Seems redundant to me, since a GET is being made on instances. I would just call it 'name', but let's follow whatever pattern is established.

Reply:
i will update  to `name`. The naming changes as with constructing JSON manually, prone to error, but i looked at few places and `name` is used.